risedev.yml

```diff
@@ -524,6 +524,17 @@ risedev:
     - use: frontend
     - use: compactor
 
+  ci-1cn-1fe-with-recovery:
+    - use: minio
+    - use: etcd
+      unsafe-no-fsync: true
+    - use: meta-node
+      enable-committed-sst-sanity-check: true
+    - use: compute-node
+      enable-tiered-cache: true
+    - use: frontend
+    - use: compactor
```

Comment:
no settings for source/sink node? It is ok now but the should be one later

Reply:
In ci pipeline, the connector node will build from source in the build-other.sh